src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Intake extends SubsystemBase {
+    Intake INSTANCE = null;
```

Comment:
private static por favor

Reply:
done.

Reply:
Insert the commit hash por favor

Reply:
17d58bd04bb7027af45f55816b6a1e73e5e7f7b2